csrc/index_compute.cpp

```diff
@@ -3338,18 +3338,19 @@ Val* Index::eye(
 // To schedule TMA, the overall process is:
 //
 // First, we need to specify how TMA should view the tensor. In the above
-// diagram, TMA view the tensor as a 2D tensor [I1, I2]. Note that the
-// IterDomain expressions between the allocation domain and [I1, I2] must be
+// diagram, TMA view the tensor as a 3D tensor [I0, I1, I2]. Note that the
+// IterDomain expressions between the allocation domain and [I0, I1, I2] must be
 // compatible with the allocation domain, for example, we can not merge
 // discontiguous IterDomains, and we can not have indivisible splits either.
-// We call [I1, I2] the "TMA domain" of the tensor.
+// We call [I0, I1, I2] the "TMA domain" of the tensor.
 //
 // Second, we need to tile each dimension of the TMA domain of the tensor. If we
 // want a contiguous box dim (see Definition 3: "boxId" below), then we do one
 // split to get the box. If we want a strided box dim, then we do two splits,
 // the first split is to get the box, and the second split is to get the stride.
-// In the above diagram, I1 has a contiguous box dim, and I2 has a strided box
-// dim.
+// Note that if the tile size is 1, we can skip tiling, for this case, we call
```

Comment:
To clarify, does this mean there's only one tile or there may be multiple tiles, each of which is a size-1 tile? I suppose it's the former.

Reply:
I am referring to the later. There can be multiple tiles, but the tile size must be one. For this case, we do not need to create a Bulk ID. For the former case, where there is only one tile, what we need is to parallelize the entire dim as `Bulk`, instead of doing a split and parallelize the inner as `Bulk`. The former case is done in https://github.com/NVIDIA/Fuser/pull/2004, which is currently WIP.

Reply:
Hmm, does the size-1 tile mean a 1D tile with dimension being 1 or just a 1D tile? It should be the latter since otherwise there's no point of TMA, right?

Reply:
Oh, I see the confusion here. What I want to say is, "if the size of a box dimension is 1". So it is not necessarily a 1D tile. It can be a tile like, for example, `(1, 4)`.

Reply:
For example, without this PR, if I have a tensor of shape `[1024, 1000, 128]`, and I want to load with tile size `[1, 16, 4]`, then I have to schedule the fusion like `[1024/1, 1 (bulk), 1000/16, 16 (bulk), 128/4, 4 (bulk)]`. With this PR, I can omit the first split, and leave the fusion as `[1024, 1000/16, 16 (bulk), 128/4, 4 (bulk)]`

Reply:
Thanks. That clarifies my confusion indeed. I think it'd be more clear to say "a dimension of a tile is 1".

Reply:
I changed it to `a box dimension has size 1` in https://github.com/NVIDIA/Fuser/pull/2004 in https://github.com/NVIDIA/Fuser/pull/2004/commits/095101f267109ef52cd8a2a78d46c7f5e5e0e5e3.
I will try to leave this PR as-is (as long as there is no functional error), and resolve all PR reviews in https://github.com/NVIDIA/Fuser/pull/2004. This way, I don't need to deal with merge conflicts.